### ISSUE TITLE
 OMEGATrackinMeasures: un-selection of trajectories on TE, trajectories on TS and segments on TM should be sinked

### ISSUE THREAD

**Issue by strambc** [OmegaProject/Omega#325]
Inherits from #314 

When a trajectory is un-checked on TE-TB, or TS-TB and when segment is uncheked on TM-TB the un-checking should synch across TB views.

Currently works

**Comment by strambc** [comment]
## Step 1
![new issue_from 314_step 1](https://user-images.githubusercontent.com/4050585/32127271-a0030f06-bb43-11e7-81cf-229ff7b47cdb.png)

**Comment by strambc** [comment]
## Step 2
![new issue_from 314_step 2](https://user-images.githubusercontent.com/4050585/32127279-ab7c2e80-bb43-11e7-962d-4b39c2eca845.png)

**Comment by strambc** [comment]
## Step 3
![new issue_from 314_step 3](https://user-images.githubusercontent.com/4050585/32127285-bb212bba-bb43-11e7-9a7b-4b5e580de7df.png)

**Comment by strambc** [comment]
## Step 4
![new issue_from 314_step 4](https://user-images.githubusercontent.com/4050585/32127289-c3ef8890-bb43-11e7-8d82-3b20be87935d.png)

**Comment by Akramhar** [comment]
This is how its supposed to work.
Un check on segment propagates on segments.
Un check on trajectories propagates on trajectories.
They are 2 distinct level of visibility.

**Comment by strambc** [comment]
@Akramhar
ok then this should be changed for beta 2.0 beause the way it works now is confusing for users.
I get that segments and trajectories programmatically are different but in reality they are one derived from the other so it MAKES TOTAL SENSE for visibility of one to be transated to the other.

**Comment by Akramhar** [comment]
Not really, because if you hide 1 you may still want to see other segments, so to propagate to the track would be wrong.
On the other way if you hide a track, all segments are gonna be hidden automatically, but their visibility flag would not be changed. Doing so if you un hide the track you still have the same visibility settings you set before.
